deco-sites/otica-isabela
# add stopwatch into product-card

## What is this contribution about?

Created stopwatch for product-card promotion.

## How to test it?

Go to HomePage and see any products within shelves that might have promotions enabled.
